analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -120,7 +120,7 @@ class Pip(
                     workingDir = workingDir,
                     definitionFile = definitionFile,
                     pythonVersion = pythonVersionOption.replace(".", ""),
-                    operatingSystem = operatingSystemOption
+                    operatingSystem = if (operatingSystemOption == "mac") "macos" else operatingSystemOption
```

Comment:
"mac" is just a configuration key that is passed to `python-inspector` as-is. So instead of mapping it here, we should simply change lines 49 and 61 to say "macos" instead of "mac" to documents that's what we're expecting the user to use as a name.

Reply:
fixed.

Reply:
Thanks @kwmyoo. Now please squash all commit in this PR into one, in particular to get rid of the merge commit (use rebase instead of merge).

Reply:
@sschuberth How do I do that? Sorry, I have never made a PR for an open source project before.

Reply:
The general approach is described e.g. [here](https://stackoverflow.com/a/5190323/1127485). In your case, it would be `HEAD~3` (instead of `HEAD~12`) to squash the last 3 commits into one. When being prompted to edit the commit message, just keep the "middle" one and delete all the rest.

Reply:
Thanks for the guide. I have applied the changes. @sschuberth

Reply:
Unfortunately, editing the commit message went a bit wrong @kunwyoo. The commit message now literally reads

```
    Author: Michael Yoo <vbtkdpf148@gmail.com>
    Date:   Mon Jan 9 16:24:14 2023 +0900

        Pip: fix operating system name for macOS

        The code currently inputs operatingSystem of macOS as "mac" when running python-inspector.

        However, python-inspector expects the option of macOS to be "macos".

        Signed-off-by: Michael Yoo <vbtkdpf148@gmail.com>
```

but it should read

```
Pip: fix operating system name for macOS

The code currently inputs operatingSystem of macOS as "mac" when running
python-inspector.

However, python-inspector expects the option of macOS to be "macos".

Signed-off-by: Michael Yoo <vbtkdpf148@gmail.com>
```

So, `Pip: fix operating system name for macOS` must be the first line, `Signed-off-by: Michael Yoo <vbtkdpf148@gmail.com>` must be the last line, none of the lines should be indented, and lines should be hard-wrapped at column 75. Just do `git commit --amend`, copy & paste my message from above, and *force*-push once more.

Reply:
done.

Reply:
Sorry, there's yet more: Now the DCO check fails because you've used another email address to sign-off the commit (`vbtkdpf148@gmail.com`) than for creating the the commit (`kunwyoo@ncsoft.com`). Please align these.

Reply:
Done (for real this time)